webapp/src/js/controllers/inbox.js

```diff
@@ -662,7 +663,9 @@ var feedback = require('../modules/feedback'),
         // initialised yet
         try {
           $(this).select2('close');
-        } catch (e) {}
+        } catch (e) {
+          $log.error('Error', e);
```

Comment:
Are we sure we want to throw an error here? Or should we just document why we had an empty block before?